docs/api-testing/load-testing.md

```diff
@@ -111,6 +122,11 @@ Click any of the measurement labels to remove them from the display.
 
 <img src={useBaseUrl('/img/api-testing/load-test-report-status-codes.png')} alt="Status codes graph" width="400"/>
 
+Once the test is completed, at the bottom of the report, "Failures" section will be shown. In this section, you will see all the most recent failures that happened during the test. The section shows one row for each test execution. Each row has the date and time, the number of failures, and a link to the specific report for that test. 
+The **Load more failures** allows you to load more failures. 
```

Comment:
```suggestion
```
This last sentence is a bit redundant. I suggest we remove it.

Reply:
@lysannep by default we show only 20 failures, the "load more failures" button allows user to load more results (in groups of 20)